sklearn/pipeline.py

```diff
@@ -481,7 +481,7 @@ def _inverse_transform(self, X):
         return Xt
 
     @if_delegate_has_method(delegate='_final_estimator')
-    def score(self, X, y=None):
+    def score(self, X, y=None, **score_params):
```

Comment:
If we pass **score_params to score function people will start using it even if it is not documented, why should we do that?